Snakemake/config/ancestral_Eddie.yaml

```diff
@@ -1,5 +1,5 @@
-raw_vcf: "/home/v1jobste/HLab/share/Snakemake/Data/RawVCF/Combined_ReducedSamples1.vcf.gz"
-aligned_focal: "/home/v1jobste/HLab/share/Snakemake/Data/testAligned.txt"
+raw_vcf: "/exports/eddie/scratch/hbecher/tree_seq_pipeline/TestDataBee/RawVCF/Combined_ReducedSamples1.vcf.gz"
+aligned_focal: "/exports/eddie/scratch/hbecher/tree_seq_pipeline/TestDataBee/testAligned.txt"
 no_estsfs_chunks: 3
-estsfs_config: "/home/v1jobste/HLab/share/Snakemake/Data/config-kimura_3o.txt"
-estsfs_seed: "/home/v1jobste/HLab/share/Snakemake/Data/seedfile.txt"
+estsfs_config: "/exports/eddie/scratch/hbecher/tree_seq_pipeline/TestDataBee/config-kimura_3o.txt"
+estsfs_seed: "/exports/eddie/scratch/hbecher/tree_seq_pipeline/TestDataBee/seedfile.txt"
```

Comment:
@hannesbecher didn't you say it would be better to specify "homeDir" somewhere at the top and then use this variables - so that we don't need to change this path all the time (if all files are in the same folder)?

Reply:
Correct, I'll do that next! Wanted to make sure this is OK and working with the new VCF folder (see below).

Reply:
Custom output dir is implemented now. To be specified in `tsinfer_Eddie.yaml`:  
`o_dir: /exports/eddie/scratch/hbecher/beeTestOut`  
Can also be a path relative to the position of the Snakefile